scripts/amq-chown/install.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+set -e
+
+chmod 0755 $AMQ_HOME
+chmod 0755 $AMQ_HOME/bin
+chown -R jboss:root $AMQ_HOME
+chmod -R g+rwX $AMQ_HOME
+
+echo "#!/bin/sh" > $AMQ_HOME/runscript.sh
+echo "$AMQ_HOME/bin/artemis create $AMQ_HOME/broker" >> $AMQ_HOME/runscript.sh
+echo "$AMQ_HOME/broker/bin/artemis run" >> $AMQ_HOME/runscript.sh
+chmod 0755 $AMQ_HOME/runscript.sh
```

Comment:
Just create the `runscript.sh` file with the proper content and add it to the image and copy in the `install.sh` script to the correct location in the image.